portfolio/src/main/webapp/assets/script.js

```diff
@@ -43,3 +18,126 @@ function collapseMenu() {
         });
     }
 }
+
+/**
+ * Opens and closes the navigation bar.
+ */
+
+function openNav() {
+    document.getElementById("nav").style.width = "200px";
+    document.getElementById("nav-main").style.marginLeft = "200px";
+}
+
+function closeNav() {
+    document.getElementById("nav").style.width = "0px";
+    document.getElementById("nav-main").style.marginLeft = "0px";
+}
+
+/** 
+ * Buttons for photo slider transitions, indexers.
+ */
+
+var index = 1;
+
+function move(n) {
+    show(index += n);
+}
+
+function current(n) {
+    show(index = n);
+}
+
+function show(n) {
```

Comment:
This function is doing enough things where I feel it deserves a function description.